docs/topics/articles.md

```diff
@@ -0,0 +1,184 @@
+# `Article`s and `ListingPage`s
+
+`Article`s are publishable content types that live in collections. Normally
+articles are mounted under a parent's URL.
+
+An example of an article is a Press Release, which would be mounted under a
+Press Release Listing Page. The Listing Page has the URL "/press" and thus all
+the Press Releases have the URL "/press/<slug>".
+
+## `ArticleBase`
+
+The `icekit.articles.models.ArticleBase` inherits `title`, `slug` and
+publishing fields, and each `ArticleBase` subclass implements some key
+functionality:
+
+1. **Links to parents.**
+In order for an Article to know its URL (which is based on a parent's URL), it
+should define a `parent` link, which is normally a link to a `ListingPage`.
```

Comment:
The Press Release behaviour you describe is already implemented: https://github.com/ic-labs/icekit-press-releases/blob/master/press_releases/models.py#L76

Given the watch/read/listen from SFMOMA, how would you implement a listing page at `/watch/` that lists a subset, and to serve that subset at `/watch/<slug>` ? It seems you're either hardcoding the URL or using a database link. My approach allows the latter, without precluding the former.